dev/org.eclipse.codewind.openapi.ui.test/src/org/eclipse/codewind/openapi/ui/test/wizard/WizardTest.java

```diff
@@ -0,0 +1,86 @@
+/*******************************************************************************
+ * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.codewind.openapi.ui.test.wizard;
+
+import org.eclipse.codewind.openapi.test.Constants;
+import org.eclipse.codewind.openapi.ui.test.BaseTestCase;
+import org.eclipse.codewind.openapi.ui.test.UIConstants;
+import org.eclipse.core.resources.ResourcesPlugin;
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.core.runtime.IProgressMonitor;
+import org.eclipse.core.runtime.NullProgressMonitor;
+import org.eclipse.swtbot.eclipse.finder.SWTWorkbenchBot;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.junit.runners.MethodSorters;
+
+@RunWith(JUnit4.class)
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+public class WizardTest extends BaseTestCase {
+
+	public WizardTest() {
+		this.newProjectName = "ClientStubPHPProject";
+	}
+	
+	@BeforeClass
+	public static void beforeClass() {
+		SWTWorkbenchBot swtWorkbenchBot = new SWTWorkbenchBot();
+		try
+		{
+			swtWorkbenchBot.viewByTitle("Welcome").close();
+		} catch (Exception e) {
+			
+		}
+	}
+
+	@Test
+	public void testWorkingYAMLPHPCLient01() {
+		sourceDefinition = Constants.PETSTORE_30;
```

Comment:
I set this up so the project name can be customized too for each `@Test`.  So, the `sourceDefinition` will be copied to the `targetDefinitionInProject` to test that the file name itself doesn't matter for enabling the context menu.  The other is the `newProjectName`.  Even though it doesn't affect the test results, it would be nice to have the project name related to the wizard test.  So for example, anywhere on lines 52 to 53, before setup(), add:

`this.newProjectName = "ClientStubPHPProject";`

Line 60-61, add:  `this.newProjectName = "ServerStubNodeProject";`

Line 70-71, add: `this.newProjectName = "genHtmlProject";`

When I ran the tests, I 'visually' saw that the same `ClientStubPHPProject` project name was used for each of the tests.

Reply:
addressed